dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -23,6 +36,53 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationTextProperties SetMaxLines(Int32 value);
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationTextStyle
+    {
+        Default,
+        Caption,
+        CaptionSubtle,
+        Body,
+        BodySubtle,
+        Base,
+        BaseSubtle,
+        Subtitle,
+        SubtitleSubtle,
+        Title,
+        TitleSubtle,
+        TitleNumeral,
+        Subheader,
+        SubheaderSubtle,
+        SubheaderNumeral,
+        Header,
+        HeaderSubtle,
+        HeaderNumeral,
+    };
+
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationTextAlign
+    {
+        Default,
+        Auto,
+        Left,
+        Center,
+        Right,
+    };
+
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationExtendedTextProperties : AppNotificationTextProperties
+    {
+        AppNotificationExtendedTextProperties();
+
+        AppNotificationTextStyle Style;
+        Int32 MinLines;
```

Comment:
Why not unsigned?

MinLines=-666 means something?

Reply:
good point, that could be unsigned.